src/deploy-to-network.ts

```diff
@@ -0,0 +1,162 @@
+}
+
+export async function testEns(): Promise<void> {
+  console.log("The wallet address will become the owner of the 'eth' domain");
+  const provider = await walletConnectProvider();
+  const signer = provider.getSigner();
+  const signerAddr = await signer.getAddress();
+  const ens = await deploy.deployEns(signer);
+  printDeployed("ENS", ens);
+  await deploy.setSubnodeOwner(ens, "", "eth", signerAddr);
+  console.log("Registered domain 'eth' for", signerAddr);
+  await provider.close();
+}
+
+export async function phase0(): Promise<void> {
+  console.log("The wallet address must own the 'eth' domain in the ENS");
+  const provider = await walletConnectProvider();
+  const signer = provider.getSigner();
+  const govGuardian = await askForAddress("of the governor guardian");
+  const tokensHolder = await askForAddress("to hold all the Radicle Tokens");
+  const ensAddr = await askForAddress("of the ENS");
+
+  const token = await deploy.deployRadicleToken(signer, tokensHolder);
+  printDeployed("Radicle Token", token);
+
+  const govAddr = utils.getContractAddress({
+    from: await signer.getAddress(),
+    nonce: 1 + (await signer.getTransactionCount()),
+  });
+  const delay = 2 * 60 * 60 * 24;
+  const timelock = await deploy.deployTimelock(signer, govAddr, delay);
+  printDeployed("timelock", timelock);
+
+  const gov = await deploy.deployGovernance(
+    signer,
+    timelock.address,
+    token.address,
+    govGuardian
+  );
+  printDeployed("governance", gov);
+
+  const treasury = await deploy.deployTreasury(signer, govAddr);
+  printDeployed("treasury", treasury);
+
+  const ens = ENS__factory.connect(ensAddr, signer);
+  const registrar = await deploy.deployRegistrar(
+    signer,
+    constants.AddressZero, // oracle not used yet
+    constants.AddressZero, // exchange not used yet
+    token.address,
+    ens,
+    "eth",
+    "radicle",
+    govAddr
+  );
+  printDeployed("registrar", registrar);
+
+  await provider.close();
+}
+
+export async function vestedTokens(): Promise<void> {
+  console.log("The wallet owner will be the one providing tokens for vesting");
+  const provider = await walletConnectProvider();
+  const signer = provider.getSigner();
+  const token = await askForAddress("of the token");
+  const owner = await askForAddress("of the owner of all created vestings");
+  const vestingPeriod = await askForBigNumber(
+    "vesting period of all created vestings in seconds"
+  );
+  const cliffPeriod = await askForBigNumber(
+    "cliff period of all created vestings in seconds"
+  );
+  do {
+    const beneficiary = await askForAddress("of beneficiary");
+    const amount = await askForBigNumber("amount to vest");
+    const vestingStartTime = await askForBigNumber(
+      "vesting start time in seconds since epoch"
+    );
+    await deploy.deployVestingToken(
+      signer,
+      token,
+      owner,
+      beneficiary,
+      amount,
+      vestingStartTime,
+      vestingPeriod,
+      cliffPeriod
+    );
```

Comment:
We should print the address here of each vesting contract

Reply:
Fixed